lib/src/main/java/uk/gov/di/ipv/core/library/domain/ContraIndicator.java

```diff
@@ -1,20 +1,25 @@
 package uk.gov.di.ipv.core.library.domain;
 
 import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
-import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
 
 import java.time.Instant;
 import java.util.List;
 
 @Getter
-@Builder
-@ExcludeFromGeneratedCoverageReport
-public class ContraIndicator {
+@Builder(toBuilder = true)
+@EqualsAndHashCode
```

Comment:
We don't need the equals, hashcode, and compareTo - nor does ContraIndicator need to implement Comparable since `ContraIndications.getLatestContraIndicator` supplies a Comparator on issuanceDate directly, hence ContraIndicator doesn't need the automatic ordering that we were giving it. Simplifies/eliminates the ContraIndicator tests as well.